migration/src/m0000350_add_sbom_file.rs

```diff
@@ -0,0 +1,44 @@
+use crate::m0000030_create_sbom::SbomNode;
```

Comment:
I know it feels like repetition, but let's try for a while to put enums locally and not import from other mgirations.